lib/sources/WebStreamFileSource.ts

```diff
@@ -39,6 +39,8 @@ export class WebStreamFileSource implements FileSource {
 
   private _done = false
 
+  private _stream: ReadableStream
```

Comment:
This property doesn't seem used. Can it be removed?

Reply:
this property has been removed.